src/shibuya.js

```diff
@@ -28,24 +28,24 @@ const main = async () => {
     const proxyOnShibuya = shibuyaHelper.getProxyAccount(TURING_PARA_ID, shibuyaAddress);
 
     console.log(`\na) Add a proxy of Turing (paraId:${TURING_PARA_ID}) for Alice on Shibuya ...\n Proxy address: ${proxyOnShibuya}\n`);
```

Comment:
The addProxy is supposed to be a one-time call. Always check the existence of a proxy first. If there is one, skip this step.